### DISCUSSION TITLE
LSTM based recipe

### DISCUSSION THREAD

**Discussion by pratikkumar018** [espnet/espnet#4601] [Q&A] [answered]
Hi Team, 
Is there any LSTM based recipe. In the recipe files I can see all based on Transformers/conformers only.

**Comment by kan-bayashi** [comment, marked as answer]
Which task do you want?

**Reply by pratikkumar018** [reply]
I am looking for ASR task.

**Reply by kan-bayashi** [reply]
https://github.com/espnet/espnet/blob/0773b51d4e3d56a1ed569da860415c92d9e735ae/egs2/laborotv/asr1/conf/tuning/train_asr_rnn.yaml
You can find via:
https://github.com/espnet/espnet/search?q=*train_asr_rnn*&type=

**Reply by pratikkumar018** [reply]
Is it a streaming or non streaming recipe?

**Reply by kan-bayashi** [reply]
streaming config may include "streaming" in the name like:
https://github.com/espnet/espnet/blob/36653907ad1e00363d2bc1266aa7fc86660f7ff1/egs2/aishell/asr1/conf/train_asr_streaming_conformer.yaml
Otherwise, non-streaming.

**Reply by pratikkumar018** [reply]
Thanks.
